.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can you remind me why we need prettier and how it is different from `.eslintrc`?